### PR TITLE
Separate /groups between global and room

### PR DESCRIPTION
**Pull request by inactiveuser**
I noticed that the current command when broadcasted it quite floody, perhaps it would be a good idea to separate them?

I talked this over with panpawn

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1045,27 +1045,37 @@ exports.commands = {
...
 			"&amp; <b>Global Leader</b> - The above, and they can promote to global moderator and force ties<br />" +
-			"~ <b>Global Administrator</b> -  They can do anything, like change what this message says"
+			"~ <b>Global Administrator</b> - The above, and they can promote up to global administrator and can do anything"
 		);
 	},
-	groupshelp: ["/groups - Explains what the symbols (like % and @) before people's names mean.",
-		"!groups - Shows everyone that information. Requires: + % @ * # & ~"],
+	groupshelp: ["/globalgroups - Explains what the symbols (like % and @) before people's names mean.",
+		"!globalgroups - Shows everyone that information. Requires: + % @ * # & ~", "/roomgroups - Explains the list of room ranks.", 
+		 "!roomgroups - Shows everyone that information. Requires: + % @ * # & ~ ],
```
Comment (panpawn) [inline chat-plugins/info.js]: I think you're trying to remove this line, but leave the quote mark at the start of it...?

Comment (InactiveUser) [inline chat-plugins/info.js]: Fixed

File: chat-plugins/info.js [hunk 2]
```diff
@@ -1045,27 +1045,37 @@ exports.commands = {
...
 			"@ <b>Global Moderator</b> - The above, and they can globally ban users<br />" +
 			"* <b>Global Bot</b> - Like Moderator, but makes it clear that this user is a bot<br />" +
 			"&amp; <b>Global Leader</b> - The above, and they can promote to global moderator and force ties<br />" +
-			"~ <b>Global Administrator</b> -  They can do anything, like change what this message says"
+			"~ <b>Global Administrator</b> - The above, and they can promote up to global administrator and can do anything"
 		);
 	},
-	groupshelp: ["/groups - Explains what the symbols (like % and @) before people's names mean.",
-		"!groups - Shows everyone that information. Requires: + % @ * # & ~"],
+	groupshelp: ["/globalgroups - Explains what the symbols (like % and @) before people's names mean.",
```
Comment (panpawn) [inline chat-plugins/info.js]: The format for multiline help commands are like this:
```
blahhelp: [
     "blah line 1",
     "blah line 2",
],
```

(replace the five spaces with an indent blah blah blah)

File: chat-plugins/info.js [hunk 3]
```diff
@@ -1045,27 +1045,37 @@ exports.commands = {
...
-			"# <b>Room Owner</b> - They are leaders of the room and can almost totally control it<br /><br />" +
-			"<b>Global Rank</b><br />" +
+			"<b>Global Ranks:</b><br />" +
 			"+ <b>Global Voice</b> - They can use ! commands like !groups, and talk during moderated chat<br />" +
 			"% <b>Global Driver</b> - The above, and they can also lock users and check for alts<br />" +
 			"@ <b>Global Moderator</b> - The above, and they can globally ban users<br />" +
 			"* <b>Global Bot</b> - Like Moderator, but makes it clear that this user is a bot<br />" +
 			"&amp; <b>Global Leader</b> - The above, and they can promote to global moderator and force ties<br />" +
-			"~ <b>Global Administrator</b> -  They can do anything, like change what this message says"
+			"~ <b>Global Administrator</b> - The above, and they can promote up to global administrator and can do anything"
```
Comment (panpawn) [inline chat-plugins/info.js]: I think the old message is better here.

File: chat-plugins/info.js [hunk 4]
```diff
@@ -1045,27 +1045,36 @@ exports.commands = {
...
 			"* <b>Global Bot</b> - Like Moderator, but makes it clear that this user is a bot<br />" +
 			"&amp; <b>Global Leader</b> - The above, and they can promote to global moderator and force ties<br />" +
-			"~ <b>Global Administrator</b> -  They can do anything, like change what this message says"
+			"~ <b>Global Administrator</b> - They can do anything, like change what this message says"
 		);
 	},
-	groupshelp: ["/groups - Explains what the symbols (like % and @) before people's names mean.",
-		"!groups - Shows everyone that information. Requires: + % @ * # & ~"],
+	groupshelp: ["/globalgroups OR /roomgroups - Explains what the symbols (like % and @) before people's names mean.",
+-		"!globalgroups OR !roomgroups - Shows everyone that information. Requires: + % @ * # & ~"],
```
Comment (bumbadadabum) [inline chat-plugins/info.js]: When you fix the build this is fine in my opinion.

Comment (panpawn) [inline chat-plugins/info.js]: According to https://github.com/Zarel/Pokemon-Showdown/commit/4fa6c5499bc1530a58c27cdbc6c82a0d34024369, it seems as though multi-line arrays shouldn't be formatted like this, including multi-line help commands.

Comment (panpawn) [inline chat-plugins/info.js]: Plus, standards are a good thing

**Comment by InactiveUser** [comment]
whew, finally. Thanks for the tips

**Comment by Zarel** [comment]
Would rather leave the default as it currently is. Maybe a `/groups global` or `/groups room` to filter.

**Pull request closed without merging**